src/core_modules/capture-core/components/Pages/MainPage/EventWorkingListsInit/Header/workingListsInitHeader.types.js

```diff
@@ -0,0 +1,2 @@
+// @flow
+export type Props = $ReadOnly<{}>;
```

Comment:
Also here perhaps annotate using the exact thingy if you feel it worths the efford